doc/userguide/rules/smtp-keywords.rst

```diff
@@ -18,6 +18,67 @@ Signature Example:
 
 For additional information on the ``file.name`` keyword, see :doc:`file-keywords`.
 
+
+smtp.helo
+---------
+
+SMTP helo is the parameter passed to the first HELO command from the client.
```

Comment:
since this is implemented as a tx keyword, but takes the helo from the state instead of the tx, we could see an alert per tx for this right?

Perhaps it is worth exploring if the helo should go in it's own tx?

Reply:
On 2nd thought, probably good to keep it as is, as it would be helpful to mix it in with the per tx keywords.

Reply:
I think the doc needs to state that a rule with this keyword can match more than once per flow, even the helo occurs only once in the flow.

Reply:
right, thanks, anything else to change ?

Reply:
no other comments from me, looks good